src/components/entity-panel/sections/team/TeamListItem.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <v-list-item
+    :class="[' ml-2 px-0', active ? 'custom-team-item-active' : '']"
+    @click="$emit('click', entity)"
+  >
+    <v-list-item-avatar class="mx-2 my-0" size="30">
+      <img :style="{ backgroundColor: entity.color }" :src="entity.image" class="custom-team-image" />
+    </v-list-item-avatar>
+    <v-list-item-content>
+      <v-list-item-title class="custom-team-title">
+        <span v-if="entity.alias" v-text="entity.alias" />
+        <span>
+          <span v-text="entity.name" />
+          <v-chip v-if="entity.tier" small>T-{{ entity.tier }}</v-chip>
+        </span>
+      </v-list-item-title>
+    </v-list-item-content>
+    <v-list-item-action class="pa-0 ma-0" @click.stop="teamListItemActionHandler">
+      <v-menu
+        offset-y
+        nudge-left="100"
+        nudge-width="80"
+        content-class="elevation-2"
+      >
+        <template v-slot:activator="{ on: menuItem }">
+          <v-btn
+            v-on="menuItem"
+            small
+            icon
+            class="mr-1"
+          >
+            <v-icon small color="grey">fa-ellipsis-v</v-icon>
+          </v-btn>
+        </template>
+        <v-card tile>
+          <v-list dense>
+            <v-list-item
+              v-for="(cardItem, index) in menuItems"
+              :key="index"
+              @click="teamMenuOnClickHandler(cardItem)"
+            >
+              <v-list-item-icon class="custom-entity-menu-icon">
+                <v-icon
+                  :color="getTeamMenuColour(entity, cardItem)"
+                  small
+                  v-text="getTeamMenuIcon(entity, cardItem)" />
+              </v-list-item-icon>
+              <v-list-item-content>
+                <v-list-item-title v-text="getTeamMenuText(entity, cardItem)" />
+              </v-list-item-content>
+            </v-list-item>
+          </v-list>
+        </v-card>
+      </v-menu>
+    </v-list-item-action>
+  </v-list-item>
+</template>
+
+<script lang="ts">
+import Component from 'vue-class-component'
+import Vue from 'vue'
+import { Prop } from 'vue-property-decorator'
+import { Ship } from '@/types/Games/Wows'
+import { TeamMenuOptions, MenuItem } from '../../types'
+import { Game } from '@/store/types'
+import { getGameName } from '@/games/utils'
+import { EventBus } from '@/event-bus'
+
+@Component({
+  name: 'TeamListItem'
+})
+export default class TeamListItem extends Vue {
+  @Prop() private readonly entity!: Ship
+  @Prop() private readonly active!: boolean
+  @Prop() private readonly game!: Game
+
+  getGameName = getGameName
+  entityName: string = this.getGameName(this.game)
+
+  teamMenuOnClickHandler (menuItem: MenuItem) {
+    switch (menuItem.action) {
+      case TeamMenuOptions.STATS:
+        EventBus.$emit('openTheEntityPropertiesModal', this.entity)
```

Comment:
Would probably be better to use here enum or something similar as refactoring this later would benefit from it.